pkg/daemon/handler.go

```diff
@@ -245,14 +245,61 @@ func (csh cniServerHandler) handleAdd(req *restful.Request, resp *restful.Respon
 			mtu = csh.Config.MTU
 		}
 
-		klog.Infof("create container interface %s mac %s, ip %s, cidr %s, gw %s, custom routes %v", ifName, macAddr, ipAddr, cidr, gw, podRequest.Routes)
+		// routes used for access from underlay to overlay
```

Comment:
what about overlay to underlay, do we need a nat when across nodes

Reply:
> what about overlay to underlay, do we need a nat when across nodes

Overlay to underlay has the same packet path with overlay to outside the cluster. Iptables rules introduced in PR #1578 allow this kind of traffic for ICMP and TCP.